mbus-api/src/v0.rs

```diff
@@ -102,6 +104,8 @@ pub enum MessageIdVs {
     AddVolumeNexus,
     /// Remove nexus from volume
     RemoveVolumeNexus,
+    /// JSON gRPC request
+    JsonGrpcRequest,
```

Comment:
nit: none of the other requests have request in the name, could it just be JsonGrpc?

Reply:
Done